torch_xla/csrc/runtime/BUILD

```diff
@@ -195,6 +195,7 @@ cc_library(
     "@org_tensorflow//tensorflow/core/distributed_runtime/rpc:grpc_runtime",
     "@org_tensorflow//tensorflow/core/protobuf/tpu:topology_proto_cc",
     "@org_tensorflow//tensorflow/compiler/xla/stream_executor:stream_executor_impl",
+    "@org_tensorflow//tensorflow/tsl/framework:allocator",
```

Comment:
what's this for?

Reply:
this dep is included before apply OpenXLA's patch in `xla_bzl.diff`, error log shows missing this dep when build, will try again without this modify, thanks, Jack

Reply:
updated in https://github.com/pytorch/xla/pull/5165/commits/fd2d9aa2ce84e5066eb1870318578f959af19b22

Reply:
this dep is needed because of https://github.com/pytorch/xla/actions/runs/5286371266/jobs/9566019205